app/src/main/java/com/jnj/vaccinetracker/register/dialogs/QrCodeGeneratorDialog.kt

```diff
@@ -0,0 +1,42 @@
+package com.jnj.vaccinetracker.register.dialogs
+
+import android.os.Bundle
+import android.util.Log
+import android.view.View
+import android.widget.Button
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.fragment.app.DialogFragment
+import com.google.zxing.BarcodeFormat
+import com.jnj.vaccinetracker.R
+import com.journeyapps.barcodescanner.BarcodeEncoder
+import kotlin.random.Random
+
+class QrCodeGeneratorDialog(private val participantId: String) : DialogFragment(R.layout.dialog_qr_code_generator) {
```

Comment:
this is strange way DialogFragment(R.layout.dialog_qr_code_generator)

please extend by BaseDialogFragment and make binding with layout in onCreateView method. 

Please take a look at our other dialogs and keep this approach.

Reply:
Resolved